rdmd.d

```diff
@@ -916,3 +898,78 @@ void yap(size_t line = __LINE__, T...)(auto ref T stuff)
     debug stderr.writeln(line, ": ", stuff);
     else stderr.writeln(stuff);
 }
+
+struct FileAttributes
+{
+    uint attributes;
+    bool exists() const { return attributes != 0xFFFFFFFF; }
+    bool isFile() const
+        in { assert(exists); } do
+    {
+        import std.file : attrIsFile;
+        return attrIsFile(attributes);
+    }
+}
+
+struct Yap
+{
+    static FileAttributes getFileAttributes(const(char)[] name)
```

Comment:
Use `static:` to make it more clear that this is a namespace and has no non-static members.

Reply:
Ok (changed in #299)